src/index.js

```diff
@@ -1,10 +1,92 @@
 #!/usr/bin/env node
 
-require('yargs')
+/**
+ * Networks available
+ */
+let networks = {
+  mainnet: 'https://mainnet.infura.io',
+  ropsten: 'https://ropsten.infura.io',
+  rinkeby: 'https://rinkeby.infura.io',
+  kovan: 'https://kovan.infura.io',
+  sokol: 'https://sokol.poa.network',
+  poa: 'https://core.poa.network',
+  local: 'http://localhost:8545'
+}
+
+let yargs = require('yargs')
   .option('url', {
     description: 'URL of the ethereum node to connect',
-    default: 'http://localhost:8545'
+    default: 'http://localhost:8545',
+    type: 'string'
+  })
+  .option('mainnet', {
+    describe: `Url of the mainnet ethereum node to connect: ${networks.mainnet}`,
+    type: 'boolean'
+  })
+  .option('ropsten', {
+    describe: `Url of the ropsten ethereum node to connect: ${networks.ropsten}`,
+    type: 'boolean'
+  })
+  .option('rinkeby', {
+    describe: `Url of the rinkeby ethereum node to connect: ${networks.rinkeby}`,
+    type: 'boolean'
+  })
+  .option('kovan', {
+    describe: `Url of the kovan ethereum node to connect: ${networks.kovan}`,
+    type: 'boolean'
+  })
+  .option('sokol', {
+    describe: `Url of the sokol ethereum node to connect: ${networks.sokol}`,
+    type: 'boolean'
   })
+  .option('poa', {
+    describe: `Url of the poa ethereum node to connect:  ${networks.poa}`,
+    type: 'boolean'
+  })
+  .option('local', {
+    describe: `Url of the local ethereum node to connect:  ${networks.local}`,
+    type: 'boolean'
+  })
+  .check(function(argv) {
+    // Mantain the amount of networks in argv
+    let networksInArgv = 0
+
+    // Check is setted the default url or not, yargs doesnt have an option for this
+    let optionUrlIsNotDefault = argv['url'] !== yargs.getOptions().default.url
+
+    // Check if the url arg is available, the url arg is string and is not default
+    if (
+      typeof argv['url'] === 'string' &&
```

Comment:
This check makes the code harder to read and doesn't really add anything. `yargs` already converts the argument to a string and, even if it didn't, the later check is enough.